samples/crypto/hmac/README.rst

```diff
@@ -7,21 +7,24 @@ Crypto: HMAC
    :local:
    :depth: 2
 
+The HMAC sample shows how to perform HMAC signing and verification operations using the SHA-256 hashing algorithm.
```

Comment:
Inconsistency. SHA-256 is a mehanism used by HMAC

"shows how to perform a signing and verification operation using the HMAC algorithm"

Reply:
Are you sure about that? I have seen many variants of the HMAC referenced, like HMAC-MD5, HMAC-SHA1 etc.

Reply:
This probably could be an additional line of text for instance: 

"The HMAC algorithm in this sample is using SHA-256"